CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- [vtex setup] Create new flags for specifying what to setup:
+  - `--all`: Select all setup flags existent.
+  - `--typings`: Download and setup GraphQL and React typings.
+  - `--tooling`: Setup Prettier, Husky and ESLint.
```

Comment:
@lucasecdb What do you think?
Since we're specifying which tooling are being installed I think it makes sense to change to `--tstooling`, or even `--ts-tooling`, which would be better @klzns?

Reply:
I think the idea would be to include dotnet tooling here, no?

Reply:
The command should be smart enough to configure the right tooling for the project 🤔

Reply:
But then there would be conflicts on the flag documentation:
`"--tooling": "Setup Prettier, Husky and ESLint for typescript. For Dotnet setup A, B..."`

Reply:
can we have multiple lines in the description? then it could be

```
--tooling    Setup tools for each builder in the app.

             Node and React apps: Prettier, Husky and ESLint
             .NET apps: Resharper, StyleCop
```

Reply:
Nice!